Topics/Development_Process/Documentation/Swagger_API_Documentation.md

```diff
@@ -0,0 +1,44 @@
+# API Documentation with SwaggerHub
+
+## Table of Contents
+- [Purpose of API Documentation](#purpose-of-api-documentation)
+- [Keys to Good API Documentation](#keys-to-good-api-documentation)
+- [Introduction of SwaggerHub](#introduction-of-swaggerhub)
+  - [Features](#features)
+- [Other Free Tool](#other-free-tool)
+
+
+## Purpose of API Documentation
+As the name suggests, API documentation describes a handbook of instructions and expectations around working with the project's APIs. 
+This allows the user to easily and accurately interact with the API.
+This is super important for projects with different people who are creating and using the API, including a front-end and backend team, your group and your partner, or even you and future you!
+
+## Key to Good API Documentation
+- proofread for correct spelling and grammar 
+- ensure the technical background needed to understand is reasonable
+- cover all the basics of each endpoint:
+  - the path of each endpoint
+  - what methods each endpoint supports
+  - what is accepted and/or what is returned from each endpoint
+    - including sample responses and errors
+- update the documentation when changes are made to the code
+- consistent format to allow easy access to relevant information
+
+## Introduction of SwaggerHub
+[SwaggerHub](https://swagger.io/tools/swaggerhub/) is one tool created to make API documentation easy to edit and read. 
```

Comment:
if you want, you can probably expand more on this. Like why use swaggerhub over something else. What makes this special? Y not google doc

Reply:
wait never mind I see you expand it in the features section